Dragonflight/AlgetharAcademy/Locales/esES.lua

```diff
@@ -2,22 +2,22 @@ local L = BigWigs:NewBossLocale("Algeth'ar Academy Trash", "esES") or BigWigs:Ne
 if not L then return end
 if L then
 	L.custom_on_recruiter_autotalk = "Hablar automáticamente"
-	--L.custom_on_recruiter_autotalk_desc = "Instantly pledge to the Dragonflight Recruiters for a buff."
+	L.custom_on_recruiter_autotalk_desc = "Ofrece instantánemante un beneficio a los reclutadores Dragonflight"
 	L.critical_strike = "+5% Golpe crítico"
 	L.haste = "+5% Celeridad"
-	L.mastery = "+Maestría"
+	L.mastery = "+Maestría" --did you mean "+5% Maestría"?
```

Comment:
The amount of Mastery you get from the buff varies per spec and difficulty (it's a flat amount of rating), so +Mastery is the best we can do here.

Reply:
ok perfect, just tried to help in case you forgot the number.